src/concrete/ml/onnx/convert.py

```diff
@@ -176,9 +182,11 @@ def get_equivalent_numpy_forward_from_onnx(
         Callable[..., Tuple[numpy.ndarray, ...]]: The function that will execute
             the equivalent numpy function.
     """
+
+    # All onnx models should be checked, "check_model" parameter must be removed
+    # FIXME: https://github.com/zama-ai/concrete-ml-internal/issues/4157
```

Comment:
thanks for that ! as suggested initially, I think it's best if you always check , so the idea would put a warning if it's set to false (but keep this fixme) : 

```
# FIXME : ..
if not check_model:
    warnings.warn(...)
checker.check_model(onnx_model)
```

Reply:
fixed